invenio_workflows/alembic/a26f133d42a9_create_invenio_workflows_tables.py

```diff
@@ -0,0 +1,165 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Create invenio_workflows tables."""
+
+from __future__ import absolute_import, print_function
+
+import uuid
+
+from alembic import op
+from datetime import datetime
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy_utils.types import ChoiceType, UUIDType, JSONType
+from workflow.engine_db import WorkflowStatus
+from invenio_workflows.models import ObjectStatus
+
+# revision identifiers, used by Alembic.
+revision = 'a26f133d42a9'
+down_revision = '720ddf51e24b'
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
+    """Upgrade database."""
+
+    op.create_table(
+        'workflows_workflow',
+        sa.Column(
+            'uuid',
+            UUIDType,
+            primary_key=True,
+            nullable=False,
+            default=uuid.uuid4()
+        ),
+        sa.Column(
+            'name',
+            sa.String(255),
+            default='Default workflow',
+            nullable=False
+        ),
+        sa.Column('created', sa.DateTime, default=datetime.now, nullable=False),
+        sa.Column(
+            'modified',
+            sa.DateTime,
+            default=datetime.now,
+            onupdate=datetime.now,
+            nullable=False
+        ),
+        sa.Column('id_user', sa.Integer, default=0, nullable=False),
+        sa.Column(
+            'extra_data',
+            JSONType().with_variant(
+                postgresql.JSON(none_as_null=True),
+                'postgresql',
+            ),
+            default=lambda: dict(),
+            nullable=False
+        ),
+        sa.Column(
+            'status',
+            ChoiceType(WorkflowStatus, impl=sa.Integer()),
+            default=WorkflowStatus.NEW,
+            nullable=False
+        )
+    )
+
+    op.create_table(
+        'workflows_object',
+        sa.Column('id', sa.Integer, primary_key=True),
+        sa.Column(
+            'data',
+            JSONType().with_variant(
+                postgresql.JSON(none_as_null=True),
+                'postgresql',
+            ),
+            default=lambda: dict(),
+            nullable=False
+        ),
+        sa.Column(
+            'extra_data',
+            JSONType().with_variant(
+                postgresql.JSON(none_as_null=True),
+                'postgresql',
+            ),
+            default=lambda: dict(),
+            nullable=False
+        ),
+        sa.Column(
+            'id_workflow',
```

Comment:
Shouldn't this be `_id_workflow`?

Reply:
Nop, the `name` parameter overrides it ;) (why the underscore then...)

Reply:
if you check the models there is a name kwarg that renames it. Here https://github.com/inveniosoftware-contrib/invenio-workflows/blob/master/invenio_workflows/models.py#L172 .

Reply:
the `_` is needed because of the defined hybrid property below.

Reply:
👍